pre_commit_hooks/file_contents_sorter.py

```diff
@@ -65,6 +65,9 @@ def main(argv: Sequence[str] | None = None) -> int:
     )
     args = parser.parse_args(argv)
 
+    if args.ignore_case and args.unique:
+        return FAIL
```

Comment:
this "passes the test" but now silently fails with no error message

Reply:
@asottile after reviewing and getting testing working locally I settled on removing the last test in `tests/file_contents_sorter_test.py` since the new expectation would be that file contents are unmodified, the errors is printed, and the check fails. Please let me know if this is sufficient or if you would like any other modifictions. Thanks.